debian/conf/jellyfin

```diff
@@ -33,11 +33,14 @@ JELLYFIN_FFMPEG_OPT="--ffmpeg=/usr/lib/jellyfin-ffmpeg/ffmpeg"
 # [OPTIONAL] run Jellyfin without the web app
 #JELLYFIN_NOWEBAPP_OPT="--nowebclient"
 
+# Space to add additional command line options to jellyfin (for help see ~$ jellyfin --help)
+JELLYFIN_ADDITIONAL_OPTS=""
+
 #
 # SysV init/Upstart options
 #
 
 # Application username
 JELLYFIN_USER="jellyfin"
 # Full application command
-JELLYFIN_ARGS="$JELLYFIN_WEB_OPT $JELLYFIN_RESTART_OPT $JELLYFIN_FFMPEG_OPT $JELLYFIN_SERVICE_OPT $JELLYFIN_NOWEBAPP_OPT"
+JELLYFIN_ARGS="${JELLYFIN_WEB_OPT} ${JELLYFIN_RESTART_OPT} ${JELLYFIN_FFMPEG_OPT} ${JELLYFIN_SERVICE_OPT} ${JELLYFIN_NOWEBAPP_OPT} $JELLFIN_ADDITIONAL_OPTS"
```

Comment:
```suggestion
JELLYFIN_ARGS="${JELLYFIN_WEB_OPT} ${JELLYFIN_RESTART_OPT} ${JELLYFIN_FFMPEG_OPT} ${JELLYFIN_SERVICE_OPT} ${JELLYFIN_NOWEBAPP_OPT} ${JELLFIN_ADDITIONAL_OPTS}"
```

Reply:
AFAIK this doesn't work anymore after that because the variables aren't expanding anymore after that, e.g.:

`prog $ARGS` -> arg1: `arg1`, arg2: `arg2`

`prog ${ARGS}` -> arg1: `arg1 arg2`

Reply:
There is no particular reason to pick `$ARGS` over `${ARGS}` (or the other way around) here, other than consistency.

What determines argument expansion or not is the quotes.
```bash
$ FOO="1 2 3 4"
$ python3 -c 'import sys; print(sys.argv[1:])' $FOO
['1', '2', '3', '4']
$ python3 -c 'import sys; print(sys.argv[1:])' ${FOO}
['1', '2', '3', '4']
$ python3 -c 'import sys; print(sys.argv[1:])' "${FOO}"
['1 2 3 4']
$ python3 -c 'import sys; print(sys.argv[1:])' "$FOO"
['1 2 3 4']
```

The reason for why using `${ARGS}` is required in some cases, is when you want the variable up next to text like this;
```bash
$ echo $FOObar

$ echo ${FOO}bar
1 2 3 4bar
```